deploy/determined_deploy/gcp/gcp.py

```diff
@@ -28,15 +25,12 @@ def terraform_init(configs: Dict, env: Dict):
     ]
 
     command += [terraform_dir]
-    command = " ".join(command)
 
-    output = subprocess.Popen(command, env=env, shell=True, stdout=sys.stdout)
+    output = subprocess.Popen(" ".join(command), env=env, shell=True, stdout=sys.stdout)
```

Comment:
blocking: Any reason not to remove `shell=True` instead? Same thing inside `run_command`. The `terraform init` line above would have to change, but then it would match the other places below.

Reply:
I'd prefer to make the functional change in a separate PR after the fact.  If `shell=True` was ok for the last several months, I think it can be ok for another couple of weeks.  My sense is that removing `shell=True` is much higher risk of unintended side effects than the rest of the changes here.